walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java

```diff
@@ -15,9 +15,231 @@
 package com.google.sps;
 
 import java.util.Collection;
+import java.util.Collections;
+import java.util.ArrayList;
+import java.util.HashMap;
 
 public final class FindMeetingQuery {
+
+  /**
+   * This method checks if an event contains no mandatory attendees using a MeetingRequest
+   * @param request MeetingRequest object which contains list of attendees
+   * @param event The Event object's attendees are compare with the request object
+   * @return boolean if the provide event object contains no mandatory attendees
+   */
+  private boolean hasNoMandatoryAttendees(MeetingRequest request, Event event){
```

Comment:
nit: Here and below, can you add a space before the left curly brace `{`?

Reply:
Ok